src/mixins/child-apps.js

```diff
@@ -103,9 +103,10 @@ export default {
    * @method startChildApp
    */
   startChildApp(appName, options) {
-    this.getChildApp(appName).start(options);
+    const childApp = this.getChildApp(appName);
```

Comment:
This is more readable... though `return this.getChildApp(appName).start(options);` would be the same but smaller.  I'm on the fence.. you feel strongly about the `const`?